inc/Engine/Optimization/DynamicLists/IncompatiblePluginsLists/DataManager.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare(strict_types=1);
+
+namespace WP_Rocket\Engine\Optimization\DynamicLists\IncompatiblePluginsLists;
+
+use WP_Rocket\Engine\Optimization\DynamicLists\AbstractDataManager;
+
+class DataManager extends AbstractDataManager {
+
+
+	/**
+	 * Get cache transient name.
+	 *
+	 * @return string
+	 */
+	protected function get_cache_transient_name() {
+		return 'wpr_dynamic_lists_incompatible_plugins';
+	}
+
+	/**
+	 * Get lists json filename.
+	 *
+	 * @return string
+	 */
+	protected function get_json_filename() {
+		return 'dynamic-lists-incompatible-plugins';
+	}
+
+	/**
+	 * Gets the lists content
+	 *
+	 * @return array
+	 */
+	public function get_lists() {
+		$lists          = [];
+		$list_from_json = parent::get_lists();
+		foreach ( $list_from_json as $conditions => $list ) {
+			if ( $this->meet_conditions( $conditions ) ) {
+				$list  = array_column( $list, 'file', 'slug' );
+				$lists = array_merge( $lists, $list );
```

Comment:
```suggestion
				$lists[$list['slug']] = $list['file'];
```
is this a valid suggestion?

Reply:
@engahmeds3ed It didn't work, `$list` is an array of objects